src/transformers/training_args.py

```diff
@@ -524,7 +524,15 @@ class TrainingArguments:
     )
     ort: Optional[bool] = field(
         default=False,
-        metadata={"help": "Enable Ort"},
+        metadata={"help": "Enable Ort."},
+    )
+    ort_fused_adam: Optional[bool] = field(
```

Comment:
Unfortunately we cannot, because here we wrap optimizer separately with wrapping model with ORTModule. 

optimizer and model are different components, so we could not share the Debug options in current design.

Reply:
I created this PR, not aiming at merging it asap. Because ORT optimizer wrappers are still evolving, the APIs might not be mature. Put this PR here just in case anybody need run APEX O2. 

If we are ready to release those optimizer wrappers, I think we can call out this PR to be merged. 

@SherlockNoMad  @baijumeswani  @tlh20